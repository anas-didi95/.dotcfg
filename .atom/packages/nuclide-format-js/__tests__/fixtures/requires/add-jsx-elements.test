var {Component, PropTypes} = React;

var Foo = React.createClass({
  componentWillReceiveProps(props, state) {
    return shallowCompare(this, props, state);
  },

  render() {
    return (
      <div>
        <Image src={MyConstants.IMAGE_SRC} />
        <XUISingleSelector Uppercase={emptyFunction}>
          <XUISingleSelectorOption />
        </XUISingleSelector>
      </div>
    );
  },
});
