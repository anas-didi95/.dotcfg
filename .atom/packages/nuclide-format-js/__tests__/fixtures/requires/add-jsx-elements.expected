const Image = require('Image.react');
const MyConstants = require('MyConstants');
const React = require('React');
const XUISingleSelector = require('XUISingleSelector.react');
const XUISingleSelectorOption = require('XUISingleSelectorOption.react');

const emptyFunction = require('emptyFunction');
const shallowCompare = require('shallowCompare');

var {Component, PropTypes} = React;

var Foo = React.createClass({
  componentWillReceiveProps(props, state) {
    return shallowCompare(this, props, state);
  },

  render() {
    return (
      <div>
        <Image src={MyConstants.IMAGE_SRC} />
        <XUISingleSelector Uppercase={emptyFunction}>
          <XUISingleSelectorOption />
        </XUISingleSelector>
      </div>
    );
  },
});
